app/dashboard/templates/onboard.html

```diff
@@ -0,0 +1,175 @@
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+{% endcomment %}
+{% load i18n static %}
+<!DOCTYPE html>
+<html>
+  <head>
+    {% include 'shared/head.html' %}
+    {% include 'shared/cards.html' %}
+    <link rel="stylesheet" href={% static "v2/css/onboard.css" %}>
+  </head>
+  <body>
+    {% include 'shared/tag_manager_2.html' %}
+    <div class="container-fluid header dash">
+      {% include 'shared/nav.html' %}
+    </div>
+    <div class="row">
+      <div class="col-12 offset-md-3 col-md-6">
+        <form id="onboarding">
+          <div class="state">
+            <span class="step-state"></span>
+            <span class="step-state"></span>
+            <span class="step-state"></span>
+          </div>
+          <div id="step-1" class="step show block">
+            <h2>{% trans "Welcome to the Gitcoin family" %}</h2>
+            <div class="row">
+              <div class="col-12 col-md-6 left">
+                <h3>
+                  <span>{% trans "Solve real world problems" %}</span>
+                  <span>{% trans "in the blockchain / web3 ecosystem" %}</span>
+                </h3>
+                <div class="row icons">
+                  <div class="col-3">
+                    <i class="fas fa-wrench"></i>
+                    <p>{% trans "Build" %}</p>
+                  </div>
+                  <div class="col-3">
+                    <i class="fab fa-ethereum"></i>
+                    <p>{% trans "Get Paid" %}</p>
+                  </div>
+                  <div class="col-3">
+                    <i class="fab fa-connectdevelop"></i>
+                    <p>{% trans "Network" %}</p>
+                  </div>
+                </div>
+              </div>
+              <div class="col-12 col-md-6 right">
+                {% if github_handle %}
+                  <div class="logged-in">
+                    <h3>
+                      <i class="far fa-check-circle"></i>
+                      {% trans "You've logged in!" %}
+                    </h3>
+                    <div class="font-caption row">
+                      <div class="col-12">
+                        <img class="github-avatar" src="{% url "avatar" %}?repo=https://github.com/{{ github_handle }}">
+                        <span>
+                          <p>{{ github_handle }}</p>
+                          <a href="{% url "github:github_logout" %}?redirect_uri={{ request.get_full_path }}">
+                            {% trans "Logout" %}
+                          </a>
+                        </span>
+                      </div>
+                    </div>
+                  </div>
+                {% else %}
+                  <div class="logged-out">
+                    <h3>{% trans "Let's get your account setup." %}</h3>
+                    <a class="button button--primary" href="{% url "github:github_auth" %}?redirect_uri={{ request.get_full_path }}"
```

Comment:
This `url` view reference will need updated with the new views specified in `nav_auth.html`